composer.json

```diff
@@ -4,7 +4,7 @@
     "license": "LGPL-3.0+",
     "description": "Extensible and flexible API for building web forms",
     "require": {
-        "neos/flow": "*"
+        "neos/flow": "~4.0 || dev-master"
```

Comment:
As long as someone makes sure it works with the development-distribution I don't care.

But can we please **do the actual fixing** before we block this PR through this discussion?